plugins/woocommerce/src/Blocks/Patterns/PTKClient.php

```diff
@@ -14,6 +14,67 @@ class PTKClient {
 	 */
 	const PATTERNS_TOOLKIT_URL = 'https://public-api.wordpress.com/rest/v1/ptk/patterns/';
 
+	/**
+	 * The schema for the patterns toolkit.
+	 *
+	 * @var array
+	 */
+	private $schema;
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		$this->schema = [
+			'type'  => 'array',
+			'items' => [
+				'type'       => 'object',
+				'properties' => [
+					'ID'         => [
+						'type'     => 'integer',
+						'required' => true,
+					],
+					'site_id'    => [
+						'type'     => 'integer',
+						'required' => true,
+					],
+					'title'      => [
+						'type'     => 'string',
+						'required' => true,
+					],
+					'name'       => [
+						'type'     => 'string',
+						'required' => true,
+					],
+					'html'       => [
+						'type'     => 'string',
+						'required' => true,
+					],
```

Comment:
Similarly here, should we enforce a `minLength` just in case?

Reply:
Same here